### PR TITLE
Initial version for Wear

### PR DESCRIPTION
**Pull request by mauimauer**
Adds (basic) Android Wear module

### CODE REVIEW

File: app/src/main/res/layout/fragment_game.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     android:paddingRight="@dimen/activity_horizontal_margin"
     android:paddingTop="@dimen/activity_vertical_margin"
     android:paddingBottom="@dimen/activity_vertical_margin"
-    tools:context="de.stefanhoth.android.got2048.activities.GameActivity$PlaceholderFragment">
+    tools:context="de.stefanhoth.android.de.stefanhoth.android.got2048.activities.GameActivity$PlaceholderFragment">
```
Comment (stefanhoth) [inline app/src/main/res/layout/fragment_game.xml]: :question: why the double package? Can you fix that?

**Comment by mauimauer** [comment]
Yep, writing small fixes right now.

**Comment by stefanhoth** [comment]
@mauimauer cool, after that I'm happy to merge it!

**Comment by mauimauer** [comment]
Should be alright now. Ship it! We'll make Millions :D

**Comment by stefanhoth** [comment]
Ok, :ship:ing it. :+1:

**Pull request merged**